xcp_d/workflow/base.py

```diff
@@ -97,7 +99,7 @@ def init_xcpd_wf(
                 work_dir=".",
                 dummytime=0,
                 fd_thresh=0.2,
-                func_only=False,
+                process_surfaces=True,
```

Comment:
isn't false the default?

Reply:
Good point! I'll switch it to the default.